src/cmds/scripts/pbs_snapshot

```diff
@@ -74,7 +74,7 @@ else
         # Define PATH and PYTHONPATH for the users
         PTL_PREFIX=$(dirname ${__PBS_EXEC})/ptl
         if [ ! -x "${PTL_PREFIX}/lib/site-packages" ]; then
-            ptlbinpath=${__PBS_EXEC}/unsupported/fw/bin
+			ptlbinpath=${__PBS_EXEC}/unsupported/fw/bin
```

Comment:
i think you should reduce the indentation of this block, match it with the corresponding else block

Reply:
I have corrected the indentation now. Please review.